doc/source/support/frequently-asked-questions.rst

```diff
@@ -134,6 +134,37 @@ Achieving the same result in Markdown often requires raw HTML, which is less rea
 
 Switching to ``README.rst`` at Level 5 helps users appreciate the formatting power of ``.rst`` and serves as a stepping stone toward writing full documentation in ``.rst`` as part of the ``scikit-package`` documentation standard.
 
+.. _faq-set-default-prompt-value:
+
+How can I change the default values that appear in the prompt when creating projects in level 3,4,5?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For example, to change some default values prompted when creating a new package in level 5, please follow the quick steps below:
+
+1. create ``~/.skpkgrc`` by runing ``touch ~/.skpkgrc``.
+2. copy and paste the following snippets.
+
+.. code-block:: json
+
+  {
+      "default_context":
+          {
+	       "maintainer_name": "Sangjoon Lee",
+               "project_needs_c_code_compiled": ["No", "Yes"],
+	  }
+  }
+3. run ``package create public``.
+
+
+As shown above, you can enter all the level 3, 4 and 5 default values in ``default_context``. For list variables like ``project_needs_c_code_compiled``, you should only swap the order of the elements. The element in the first order will be the default value appeared in the prompt.
+
+If you want to store the configurations in a place other than ``~.skpkgrc``, add the following code to your terminal profile and place your configurations at ``$path_to_config``.
```

Comment:
``~.skpkgrc`` seems incorrect. 

And this is also "optional" and it's better to ask a question. "Would you like to store the config file in a different place" (Gives yes or no) is better than "If you want to store..."